src/utils.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2022 IBM
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#[derive(Debug, Clone)]
+pub enum SevVersion {
```

Comment:
is this for future use?
if not needed, maybe remove?

Reply:
ok nvm might as well remove.